src/road_cmd.cpp

```diff
@@ -1665,8 +1665,18 @@ static void DrawRoadBits(TileInfo *ti)
 	/* If there are no road bits, return, as there is nothing left to do */
 	if (HasAtMostOneBit(road)) return;
 
+	/* Do not draw details when invisible */
```

Comment:
Nitpick: sentences end with a dot. Please add dots at the end of your sentences in comments :)

Reply:
Thank you!